index.html

```diff
@@ -488,6 +484,15 @@ <h3>StatusList2021Credential</h3>
 This status is reversible.
                       </td>
                     </tr>
+                    <tr>
+                      <td><code>status</code></td>
```

Comment:
Maybe this should be "enumerated" or "message"?

```suggestion
                      <td><code>enumerated</code></td>
```

Reply:
Well, this would be the most general form of status associated with a thing, so I went with "status" to represent a user defined generic status messages.

Reply:
+1 to status

Reply:
this is the `statusPurpose` which in this case is to convey `status` of a credential, not necessarily instructions for whether or not the credential is valid or invalid - `message` is defined below as the array of status messages associated with status codes embedded in the bistring

Reply:
I think a significant point of confusion is that reading the term message implies " this is the message that I am sending you", but here it is used as "list of all possible message types"

I think a clearer solution could be: 
```
{
  "@context": [
    "https://www.w3.org/2018/credentials/v1",
    "https://w3id.org/vc/status-list/2021/v1"
  ],
  "id": "https://example.com/credentials/status/3",
  "type": ["VerifiableCredential", "StatusList2021Credential"],
  "issuer": "did:example:12345",
  "issued": "2021-04-05T14:27:40Z",
  "credentialSubject": {
    "id": "https://example.com/status/3#list",
    "type": "StatusList2021",
    "ttl": 500,
    "statusPurpose": "status",
    "reference": "https://example.org/status-dictionary/"
    "statusTable": [ 
        {"status":"0x0", "value":"valid", "verify": true},
        {"status":"0x1", "value":"invalid", "verify": false},
        {"status":"0x2", "value":"pending_review", "verify": true},
        ...
    ]
    "encodedList": "H4sIAAAAAAAAA-3BMQEAAADAAAAAAAAAAAAAAAAAAAAAAAAAAAIC3AYbSVKsAQAAA"
  }
}
```

Where `statusTable`  or `statusEnum` or `enumeration` is an enumeration of all possible status' that this credentialList supports